src/react/features/lenses/grid/GridView.js

```diff
@@ -7,12 +7,23 @@ import Diory from '../../../components/diories/Diory'
 import FocusDioryContainer from '../../../components/FocusDioryContainer'
 import LinkDioryContainer from '../../../components/LinkDioryContainer'
 
-const GridView = ({ diory, diorys, onDrop, onClick }) => (
+const GridView = ({ diory, diorys, reverseDiorys = [], onDrop, onClick }) => (
   <BackgroundDiory
     diory={{ id: diory.id, image: diory.image, text: diory.text }}
     gradient
     onClick={onClick}
   >
+    {reverseDiorys.map((linkDiory) => (
+      <LinkDioryContainer linkDiory={linkDiory} onDrop={onDrop} onClick={onClick}>
+        <Diory
+          key={linkDiory.id}
+          diory={linkDiory}
+          onClick={onClick}
+          elevation={2}
+          aria-controls={`panel-${linkDiory.id}`}
+        />
+      </LinkDioryContainer>
+    ))}
```

Comment:
This could have a feature test that:
1) Reverse linked diory appears to grid 
2) When linking(&creating) diory, fromDiory appears to the grid of the linked(&created) diory
3) When diory is unlinked, fromDiory disappears from the grid of the unlinked diory

If it's too hard to do 2&3, at least 1 would be nice.

Reply:
Delete tool behaves incorrectly with these reverse links.
=> it doesn't make difference if diory is reverseDiory or linkedDiory